src/peft/tuners/oft/config.py

```diff
@@ -67,9 +67,9 @@ class OFTConfig(PeftConfig):
             Whether to share the OFT parameters between blocks or not. This is `False` by default.
     """
 
-    r: int = field(default=8, metadata={"help": "OFT rank, number of OFT blocks per injected layer."})
+    r: int = field(default=0, metadata={"help": "OFT rank, number of OFT blocks per injected layer."})
```

Comment:
Let's only change the defaults if really necessary to keep the change backwards compatible. So let's leave this value at 8, `oft_block_size` at 0, and set `use_cayley_neumann` to `False`.